handshake-layer.go

```diff
@@ -29,26 +30,42 @@ const (
 //
 // TODO: File a spec bug
```

Comment:
This TODO is obsolete, if you want to kill it while you've got the patient on the table.